src/System.ComponentModel.Composition/src/System/ComponentModel/Composition/Primitives/ComposablePartException.cs

```diff
@@ -117,5 +145,25 @@ public ICompositionElement Element
         {
             get { return _element; }
         }
+
+        /// <summary>
+        ///     Gets the serialization data of the exception.
+        /// </summary>
+        /// <param name="info">
+        ///     The <see cref="SerializationInfo"/> that holds the serialized object data about the 
+        ///     <see cref="ComposablePartException"/>.
+        /// </param>
+        /// <param name="context">
+        ///     The <see cref="StreamingContext"/> that contains contextual information about the 
+        ///     source or destination.
+        /// </param>
+        /// <exception cref="ArgumentNullException">
+        ///     <paramref name="info"/> is <see langword="null"/>.
+        /// </exception>
+        public override void GetObjectData(SerializationInfo info, StreamingContext context)
```

Comment:
Technically the override should be added to the ref as well, I think, since the type isn't sealed.